CONTRIBUTING.md

```diff
@@ -95,6 +96,90 @@ have more control over the process and debug any issues.
 1. Run `rush change` to generate changelog. You will need to follow the interactive prompts.
    You can edit the added files later. If you don't add the right entries, the CI build will fail.
 
+# New linter rule promotion strategy
```

Comment:
Let's add some intro paragraph explaining what this section is about and when it is applicable.